infra/gcp/main.tf

```diff
@@ -592,30 +679,54 @@ module "hail_query" {
   storage_class = var.hail_query_bucket_storage_class
 }
 
-module "hail_test_gcs_bucket" {
-  source        = "./gcs_bucket"
-  short_name    = "hail-test"
-  location      = var.hail_test_gcs_bucket_location
+resource "random_string" "hail_test_bucket_suffix" {
+  length = 5
+}
+
+resource "google_storage_bucket" "hail_test_bucket" {
+  name = "hail-test-${random_string.hail_test_bucket_suffix.result}"
+  location = var.hail_test_gcs_bucket_location
+  force_destroy = false
   storage_class = var.hail_test_gcs_bucket_storage_class
+  lifecycle_rule {
+    action {
+      type = "Delete"
+    }
+    condition {
+      age                        = 1
+      days_since_custom_time     = 0
+      days_since_noncurrent_time = 0
+      matches_prefix             = []
+      matches_storage_class      = []
+      matches_suffix             = []
+      num_newer_versions         = 0
+      with_state                 = "ANY"
+    }
+  }
+
+  timeouts {}
 }
 
 resource "google_dns_managed_zone" "dns_zone" {
-  name = "dns-zone"
+  description = ""
+  name = "hail"
   dns_name = "hail."
   visibility = "private"
 
   private_visibility_config {
     networks {
-      network_url = google_compute_network.default.id
+      network_url = google_compute_network.default.self_link
     }
   }
+
+  timeouts {}
 }
```

Comment:
For another PR, but this DNS zone and accompanying recordset is unused and can be deleted.